data/institutions/aom-artois-mobilités.yml

```
@@ -0,0 +1,4 @@
+name: AOM Artois Mobilités
```

C'est quoi une AOM ? Ce texte est (me semble-t-il, lu aux personnes en situation de handicap).
Est ce que la suggestion te conviendrait @mjeammet  ?

```suggestion
name: Artois Mobilités
```